### PR TITLE
Add docs for auto_explain contrib module

### PR DESCRIPTION
**Pull request by dyozie**

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/modules/intro.xml [hunk 1]
```diff
@@ -13,6 +13,8 @@
         modules are installed; refer to the linked module documentation for usage
         instructions.</p>
      <ul>
+      <li><codeph><xref href="autoexplain.xml#topic_fstrm"/></codeph> provides a means for logging
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/modules/intro.xml]: the other descriptions start with a capital - i.e. **P**rovides

File: gpdb-doc/dita/ref_guide/modules/auto-explain.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE dita PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<dita>
+  <topic id="topic_fstrm">
+    <title>auto_explain</title>
+    <body>
+      <p>The <codeph>auto_explain</codeph> module provides a means for logging execution plans of
+        slow statements automatically, without having to run EXPLAIN by hand.</p>
+      <p>The Greenplum Database <codeph>pageinspect</codeph> module is based on the
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/modules/auto-explain.xml]: not sure about these references to pageinspect?  should we replace those with auto_explain?

File: gpdb-doc/dita/ref_guide/modules/modules.ditamap [hunk 3]
```diff
@@ -2,8 +2,9 @@
 <!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
 <map title="Additional Supplied Modules">
   <topicref href="intro.xml" navtitle="Additional Supplied Modules">
-    <topicref href="citext.xml" linking="none"/>
+    <topicref href="auto-explain.xml"/>
     <topicref href="dblink.xml" linking="none"/>
+    <topicref href="citext.xml" linking="none"/>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/modules/modules.ditamap]: this list was originally in alphabetically order.  how about moving citext above dblink?

File: gpdb-doc/dita/ref_guide/modules/auto-explain.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        PostgreSQL <codeph>pageinspect</codeph> module. The Greenplum version of the
+        module differs only in that it does not allow inspection of pages belonging to
+        append-optimized or external relations.</p>
+    </body>
+    <topic id="topic_reg">
+      <title>Loading the Module</title>
+      <body>
+        <p>The <codeph>auto_explain</codeph> module provides no SQL-accessible functions. To use it,
+          simply load it into the server. You can load it into an individual session by entering
+          this command as a superuser:<codeblock>LOAD 'auto_explain';</codeblock></p>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/modules/auto-explain.xml]: is the auto_explain module available only to superusers?  this statement implies it, but does not directly state it.

Comment (gfphoenix78) [reply gpdb-doc/dita/ref_guide/modules/auto-explain.xml]: Yes, non-superuser cann't load auto_explain in current implementation.

**Comment by gfphoenix78** [comment]
> is auto_explain in greenplum the same as in postgres? if so, we should state that. (see fuzzystrmatch module intro). if not, we should identify greenplum- or mpp-isms (see dblink module intro).

auto_explain in greenplum only runs in master node.

**Pull request merged**